src/index.ts

```diff
@@ -1,8 +1,164 @@
+import testData from './testData';
+import { EventsPhase } from './types/Phases';
+import {
+  Event,
+  DurationEvent,
+  CompleteEvent,
+  MetadataEvent,
+  SampleEvent,
+  ObjectEvent,
+  ClockSyncEvent,
+  AsyncEvent,
+  ContextEvent,
+  InstantEvent,
+  CounterEvent,
+  FlowEvent,
+  MemoryDumpEvent,
+  MarkEvent,
+  LinkedIDEvent,
+} from './types/EventInterfaces';
 export const sum = (a: number, b: number) => {
```

Comment:
I'd delete the test sum function and its associated test since you no longer need them. But can do this in future cleanup if needed.